lib/skate_web/controllers/version_controller.ex

```diff
@@ -0,0 +1,7 @@
+defmodule SkateWeb.VersionController do
+  use SkateWeb, :controller
+
+  def version(conn, _params) do
+    send_resp(conn, 200, Application.get_env(:skate, :version))
+  end
```

Comment:
https://github.com/mbta/skate/pull/2549/files/cf37dce0f0efb749a5224ce7f4f892e7717dd15b#r1575376527
```suggestion
  @version Application.compile_env(:skate, :version)

  def version(conn, _params) do
    send_resp(conn, 200, @version)
  end
```

Reply:
Yeah... that does mess up the tests though. I guess the value of testing this endpoint at all is pretty minimal though 🙃